app/static/js/darkmode.js

```diff
@@ -0,0 +1,31 @@
+const root = document.documentElement;
+const button = document.querySelector('#darkmode');
+let isDarkMode = (localStorage.getItem('isDarkMode') == "true");
+setThemeMode(isDarkMode, button)
+
+button.addEventListener('click', event => {
+    isDarkMode = !isDarkMode
```

Comment:
missed `;`?

Reply:
Since its optional in JavaScript, some times i miss it. But you right i need to be consistent. ill add it to all lines.